src/category_theory/limits/preserves/shapes/equalizers.lean

```diff
@@ -0,0 +1,104 @@
+/-
+Copyright (c) 2020 Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bhavik Mehta
+-/
+import category_theory.limits.shapes.equalizers
+import category_theory.limits.preserves.basic
+
+/-!
+# Preserving equalizers
+
+Constructions to relate the notions of preserving equalizers and reflecting equalizers
+to concrete forks.
+
+In particular, we show that `equalizer_comparison G f` is an isomorphism iff `G` preserves
+the limit of `f`.
+-/
+
+noncomputable theory
+
+universes v u₁ u₂
+
+open category_theory category_theory.category category_theory.limits
+
+variables {C : Type u₁} [category.{v} C]
+variables {D : Type u₂} [category.{v} D]
+variables (G : C ⥤ D)
+
+namespace preserves_equalizer
```

Comment:
Is this the best namespace to put this in?
Shouldn't it be `category_theory.*`, where may `* = equalizer`, or something like that?

Reply:
This is a good point! I think `category_theory.limits.*` is the right option since that's where all the other limits things live, I'll amend this and the others

Reply:
Looking at this again, the rest of `ct/limits/shapes/equalizers` is just in `ct.limits`, so I've put this there too